platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/ReconnectSettingsImpl.java

```diff
@@ -32,7 +32,7 @@ public class ReconnectSettingsImpl extends SubSetting implements ReconnectSettin
     /**
      * Determines what a node will do when it falls behind. If true, it will attempt a reconnect, if false, it will die.
      */
-    public boolean active = false;
+    public boolean active = true;
```

Comment:
I agree with this change, but you should make sure that it is public knowledge that this default is changing.